tests/gtest/avifreadimagetest.cc

```diff
@@ -16,43 +16,64 @@ const char* data_path = nullptr;
 
 //------------------------------------------------------------------------------
 
-bool AreSamplesEqualForAllReadSettings(const char* file_name1,
-                                       const char* file_name2) {
-  constexpr bool kIgnoreMetadata = true;
+void AreSamplesEqualForAllReadSettings(const char* file_name1,
```

Comment:
This should be renamed to `AreSamplesSimilarForAllReadSettings()` if some leeway is allowed.